README.md

```diff
@@ -1,33 +1,44 @@
-<!-- <div align="center"> -->
-<img src="https://github.com/archlinux/archinstall/raw/master/docs/logo.png" alt="drawing" width="200"/>
-
-<!-- </div> -->
 # Arch Installer
 [![Lint Python and Find Syntax Errors](https://github.com/archlinux/archinstall/actions/workflows/flake8.yaml/badge.svg)](https://github.com/archlinux/archinstall/actions/workflows/flake8.yaml)
 
+<!-- <div align="center"> -->
+   <img src="https://github.com/archlinux/archinstall/raw/master/docs/logo.png" alt="drawing" width="200"/>
+<!-- </div> -->
+
 Just another guided/automated [Arch Linux](https://wiki.archlinux.org/index.php/Arch_Linux) installer with a twist.
 The installer also doubles as a python library to install Arch Linux and manage services, packages, and other things inside the installed system *(Usually from a live medium)*.
 
-* archinstall [discord](https://discord.gg/aDeMffrxNg) server
-* archinstall [#archinstall:matrix.org](https://matrix.to/#/#archinstall:matrix.org) Matrix channel
-* archinstall [#archinstall@irc.libera.chat:6697](https://web.libera.chat/?channel=#archinstall)
-* archinstall [documentation](https://archinstall.archlinux.page/)
+### 🔗 Archinstall Community Resources
+
+- 💬 [Join the Archinstall Discord Server](https://discord.gg/aDeMffrxNg)
+- 💬 [Chat on Matrix (#archinstall:matrix.org)](https://matrix.to/#/#archinstall:matrix.org)
+- 💬 [Join the IRC Channel (#archinstall@irc.libera.chat:6697)](https://web.libera.chat/?channel=#archinstall)
+- 📚 [Read the Official Documentation](https://archinstall.archlinux.page/)
 
 # Installation & Usage
-```shell
-sudo pacman -S archinstall
-```
 
-Alternative ways to install are `git clone` the repository or `pip install --upgrade archinstall`.
+## Install 
+
+- Using pacman
+    ```shell
+    sudo pacman -S archinstall
+    ```
+- Using pip
+    ```shell 
+   pip install --upgrade archinstall
+    ```
+Alternative ways to install are `git clone` the repository.
 
 ## Running the [guided](https://github.com/archlinux/archinstall/blob/master/archinstall/scripts/guided.py) installer
 
+### if using installed
```

Comment:
I had to read this multiple times to understand it, if the git instructions are added above (they should probably include the `python -m archinstall` as it's special) there might not be a need to have these headers here at all and it can use the original `archinstall` only